src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $white-color;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    position: relative;
+    width: 100%;
+    padding-bottom: 80.72%;
```

Comment:
I think you hardcode this value from Figma, please rewrite it

Reply:
Not fixed